lib/generators/shopify_app/shop_model/shop_model_generator.rb

```diff
@@ -30,9 +30,13 @@ def rails_migration_version
         Rails.version.match(/\d\.\d/)[0]
       end
 
-      # for generating a timestamp when using `create_migration`
-      def self.next_migration_number(dir)
```

Comment:
These methods, in this form, were not private. Can you please double check that this generator works as expected now?

Reply:
These methods are test covered and failed previously but pass currently.